src/routes/my/_Empty.svelte

```diff
@@ -3,10 +3,12 @@
 </script>
 
 <div class="column hv-center txt-center">
-  <img src={$ui.nightMode ? 'empty-dark.svg' : 'empty.svg'} alt="empty" />
-  <p>Use Insights to journal your ideas, perform research, or share with others</p>
-  <br />
-  <a href="/new" class="btn-1 mrg-xl mrg--t v-center" sapper:prefetch>Add insight</a>
+  <img class="mrg--b mrg-xl" src={$ui.nightMode ? 'empty-dark.svg' : 'empty.svg'} alt="empty" />
+  <h6 class="body-1 mrg--b mrg-s">No Insights yet</h6>
+  <p class="body-2 mrg--b mrg-l">
+    Use Insights to journal your ideas, perform research, or share with others
+  </p>
+  <a href="/new" class="btn-1 v-center body-3" sapper:prefetch>Add insight</a>
```

Comment:
`body-3` is redundant, since it has a rules inherited by default.

Reply:
inherited  font size is 14px we need size 16px here